okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/BlockingLimiter.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copied from netflix/concurrency-limits 0.1.7 and modified; original copyright notice below.
+ *
+ * See {@link https://github.com/Netflix/concurrency-limits/blob/a4c78dabf0e8d64cbb9ca4566f4ea1d3c9346543/concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limiter/BlockingLimiter.java}
+ *
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.conjure.java.okhttp;
+
+import com.netflix.concurrency.limits.Limiter;
+import com.netflix.concurrency.limits.internal.Preconditions;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Optional;
+
+/**
+ * {@link Limiter} that blocks the caller when the limit has been reached.  The caller is
+ * blocked until the limiter has been released, or a timeout is reached.  This limiter is
+ * commonly used in batch clients that use the limiter as a back-pressure mechanism.
+ */
+public final class BlockingLimiter<ContextT> implements Limiter<ContextT> {
```

Comment:
Should be safe to remove this once https://github.com/Netflix/concurrency-limits/pull/91 merges